typescript/src/v0/Mcap0StreamReader.test.ts

```diff
@@ -355,17 +355,17 @@ describe("Mcap0StreamReader", () => {
   });
 
   it.each([true, false])("parses channel info in chunk (compressed: %s)", (compressed) => {
```

Comment:
Might be worth searching the code for `channel info` (with spaces) too

```suggestion
  it.each([true, false])("parses channel in chunk (compressed: %s)", (compressed) => {
```